bluesky_live/bluesky_run.py

```diff
@@ -110,12 +117,14 @@ def descriptor(self, doc):
             self.events.new_stream(name=name)
         else:
             self._streams[name].append(doc)
+        self.events.new_doc(name="descriptor", doc=doc)
         super().descriptor(doc)
 
     @_write_locked
     def resource(self, doc):
         self._ordered.append(("resource", doc))
         self.resources[doc["uid"]] = doc
+        self.events.new_doc(name="datum_page", doc=doc)
```

Comment:
Should the name be `resource` instead of `datum_page`?

Reply:
Good catch! This would have led to very exciting bugs if uncaught!

Reply:
Fix pushed.

Reply:
Good catch!